apps/ima/sections/summary-data-sections.js

```diff
@@ -17,5 +17,20 @@ module.exports = {
         omitChangeLink: true
       }
     ]
+  },
+  'claim-details': {
+    steps: [
+      {
+        step: '/exception',
+        field: 'does-exception-apply',
+        parse: (list, req) => {
+          if (!req.sessionModel.get('steps').includes('/exception')) {
+            return null;
+          }
+          return req.sessionModel.get('does-exception-apply') === 'yes' ?
+            'Yes' + '\nDetails added' : 'No';
+        }
+      } // Removal conditions 3 & 4 are included in the summary section as part of IMB-53, so have not been added here
```

Comment:
Since that section will be added as part of another ticket you don't need to add this comment